BraintreeCore/src/test/java/com/braintreepayments/api/PreferredPaymentMethodsClientResultUnitTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.braintreepayments.api
+
+import com.braintreepayments.api.PreferredPaymentMethodsResult.Companion.fromJSON
+
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class PreferredPaymentMethodsClientResultUnitTest {
+    @Test
+    fun fromJson_whenApiDetectsPayPalPreferred_setsPayPalPreferredToTrue() {
+        val json =
+            //language=json
+            """{
+                  "data": {
+                    "preferredPaymentMethods": {
+                      "paypalPreferred": true
+                    }
+                  }
+            }"""
+            //
+        val result = fromJSON(json, false)
+        assertTrue(result.isPayPalPreferred())
+    }
+
+    @Test
+    fun fromJson_whenApiDetectsPayPalNotPreferred_setsPayPalPreferredToFalse() {
+        val json =
+            //language=json
+            """{
+                  "data": {
+                    "preferredPaymentMethods": {
+                      "paypalPreferred": false
+                    }
+                  }
+            }"""
+            //
+        val result = fromJSON(json, false)
```

Comment:
```suggestion
        val result = PreferredPaymentMethodsResult.fromJSON(json, false)
```

Reply:
It'll be better for readability since we have a ton of `fromJSON` methods in this repo.